src/functionHandlers/notarisePdt/validateInputs/validateDocument.ts

```diff
@@ -53,3 +53,16 @@ export const validateDocument = async (
   const validDomain = await isAuthorizedIssuer(issuerDomain);
   if (!validDomain) throw new UnrecognisedClinicError(issuerDomain);
 };
+
+export const validateV2Document = async (
+  attachment: WrappedDocument<HealthCertDocument>
+) => {
+  if (!attachment.data?.version)
+    throw new DocumentInvalidError(
+      "Document should include 'version' attribute"
+    );
+  const documentType = (attachment.data?.type ?? "").toUpperCase();
+  if (!attachment.data?.type || !documentType.match(/(PCR|ART)$/))
+    throw new DocumentInvalidError("Document should include 'type' attribute");
+  await validateDocument(attachment);
+};
```

Comment:
Should we also validate the other required document fields?

- `id` - Valid non-empty string
- `version` === `pdt-healthcert-v2.0`
- `validFrom` - Valid ISO 8601 datetime (e.g. `2021-08-18T05:13:53.378Z`)
- `fhirVersion` === `4.0.1`

*FYI: Only `logo` is optional. Everything else should be required.